test/pytest/test_example_scriptable_tokenzier.py

```diff
@@ -157,6 +157,7 @@ def create_mar_file(work_dir, session_mocker, jit_file_path, model_archiver):
         extra_files=os.path.join(EXAMPLE_ROOT_DIR, "index_to_name.json"),
         export_path=work_dir,
         requirements_file=None,
+        config_file=None,
```

Comment:
what's the difference btetween config_file and yaml_config_file that shows up in the torchrec test?